onadata/apps/api/permissions.py

```diff
@@ -163,6 +164,11 @@ class SubmissionReviewPermissions(XFormPermissions):
         'DELETE': ['logger.delete_xform'],
     }
 
+    def _check_is_admin_or_manager(  # pylint: disable=no-self-use
+            self, user: User, xform: XForm) -> bool:
+        return OwnerRole.user_has_role(
+            user, xform) or ManagerRole.user_has_role(user, xform)
```

Comment:
Should managers still be able to create, update & delete submission reviews?

Reply:
Yes, I think so.